src/nodes.coffee

```diff
@@ -1431,9 +1430,8 @@ exports.Param = class Param extends Base
     return @reference if @reference
     node = @name
     if node.this
-      node = node.properties[0].name
-      if node.value.reserved
-        node = new Literal o.scope.freeVariable node.value
+      name = "at_#{node.properties[0].name.value}"
```

Comment:
Names of helper functions should be counted as used if they are defined, so the conflict won't be a problem. But I can understand wanting to keep generated names starting with a single underscore for consistency, so we can leave it like this.